yoonit-camera/src/main/java/ai/cyberlabs/yoonit/camera/CameraGraphicView.kt

```diff
@@ -96,6 +112,15 @@ class CameraGraphicView constructor(
         this.faceDetectionBox = faceDetectionBox
     }
 
+    /**
+     * Draw white face detection points.
+     *
+     * @param faceLandmarks the face coordinates within the graphic view.
+     */
+    fun drawFaceLandmarks(faceLandmarks: MutableList<PointF>) {
+        this.faceLandmarks = faceLandmarks
+    }
```

Comment:
The ideia to change the function name was to be `drawFaceCotours`, right?
And this method has to draw `faceContours`. right?